packages/asset/contracts/Asset.sol

```diff
@@ -163,12 +163,12 @@ contract Asset is
         public
         view
         override(ERC1155Upgradeable, ERC1155URIStorageUpgradeable)
-        returns (string memory)
+        returns (string memory tokenURI)
     {
         return ERC1155URIStorageUpgradeable.uri(tokenId);
     }
 
-    function getTokenIdByMetadataHash(string memory metadataHash) public view returns (uint256) {
+    function getTokenIdByMetadataHash(string memory metadataHash) public view returns (uint256 tokenId) {
```

Comment:
The name return variable should also be updated in IAsset.sol for this function.

Reply:
Very true, thanks!